interfaces/IBF-dashboard/src/app/config.ts

```diff
@@ -46,8 +46,28 @@ export const LEAFLET_MAP_OPTIONS: MapOptions = {
 };
 
 export const DISASTER_TYPES_SVG_MAP = {
-  [DisasterTypeKey.floods]: '/assets/icons/Flood.svg',
-  [DisasterTypeKey.dengue]: '/assets/icons/Mosquito.svg',
-  [DisasterTypeKey.malaria]: '/assets/icons/Mosquito.svg',
-  [DisasterTypeKey.heavyRain]: '/assets/icons/Alert.svg',
+  [DisasterTypeKey.floods]: {
+    selectedTriggered: '/assets/icons/Flood_Selected_Triggered.svg',
+    selectedNonTriggered: '/assets/icons/Flood_Selected_Non Triggered.svg',
+    nonSelectedTriggered: '/assets/icons/Flood_Not Selected_Triggered.svg',
+    nonSelectedNonTriggered: '/assets/icons/Flood_Not Selected_Non Triggered.svg'
+  },
+  [DisasterTypeKey.malaria]: {
+    selectedTriggered: '/assets/icons/Malaria_Selected_Triggered.svg',
+    selectedNonTriggered: '/assets/icons/Malaria_Selected_Non Triggered.svg',
+    nonSelectedTriggered: '/assets/icons/Malaria_Not Selected_Triggered.svg',
+    nonSelectedNonTriggered: '/assets/icons/Malaria_Not Selected_Non Triggered.svg'
+  },
+  [DisasterTypeKey.dengue]: {
+    selectedTriggered: '/assets/icons/Malaria_Selected_Triggered.svg',
+    selectedNonTriggered: '/assets/icons/Malaria_Selected_Non Triggered.svg',
+    nonSelectedTriggered: '/assets/icons/Malaria_Not Selected_Triggered.svg',
+    nonSelectedNonTriggered: '/assets/icons/Malaria_Not Selected_Non Triggered.svg'
+  },
+  [DisasterTypeKey.heavyRain]: {
+    selectedTriggered: '',
```

Comment:
@SwatiTh This is not OK. This change will be implemented for all countries/disasterType, and now there is no icon for Egypt/Heavy Rainfall. Please collect this from Design and add it.